src/index.html

```diff
@@ -106,7 +119,9 @@ <h2 class="modal__title">New Post</h2>
                 placeholder="Paste a link to the picture"
                 required
               />
-              <span id="add-card-link-input-error"  class="modal__error" > Please enter a URL </span>
+              <span id="add-card-link-input-error" class="modal__error">
+                Please enter a URL
```

Comment:
Please, delete all initial error texts from `index.html` because they should appear only when the user starts typing anything in the inputs